netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -532,7 +532,8 @@ protected void onConnectionError(ChannelHandlerContext ctx, boolean outbound, Th
   @Override
   protected void onStreamError(ChannelHandlerContext ctx, boolean outbound, Throwable cause,
       StreamException http2Ex) {
-    logger.log(Level.WARNING, "Stream Error", cause);
+    Level level = http2Ex.error() == Http2Error.STREAM_CLOSED ? Level.FINE : Level.WARNING;
```

Comment:
Do we need a null check for `http2Ex` or are you sure it will never be null in this context?

Reply:
It shouldn't be null. Netty doesn't guarantee that all that well, but Netty itself assumes it is non-null and there's no reason for it to be null.